{{cookiecutter.project_slug}}/README.md

```diff
@@ -72,3 +72,62 @@ To create a superuser use the following commands:
 If React frontend was selected during the project creation
 (using our [cookiecutter](https://github.com/sixfeetup/cookiecutter-sixiedjango)), you
 can access it at [http://localhost:3000/](http://localhost:3000/).
+
+## Install Loki for log aggregation and the Kube Prometheus Stack with Grafana Dashboards
+
+### Setup AWS credentials
+
+First export the credentials to your environment variables. Change the values accordingly:
+
+```
+export AWS_ACCESS_KEY_ID='ABC123456'
+export AWS_SECRET_ACCESS_KEY='ABC123456'
+```
+
+Then create a secret in the monitoring namespace:
+
+```
+kubectl create secret generic iam-loki-s3 --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -n monitoring
+```
+
+### Install monitoring
+
+Now install the loki-stack and kube-prometheus-stack helm charts:
+
+```
+make monitoring-up
+```
+
+If you want to store the logs in an S3 bucket, you will need to include the yaml values `k8s/_monitoring/loki-stack-values.yaml` file for the `helm install loki` command in the Makefile:
+
+```
+helm install loki grafana/loki-stack --values k8s/_monitoring/loki-stack-values.yaml --namespace monitoring --create-namespace
+```
+
+### Connect to Grafana dashboard
+
+Connect to Grafana through local port forwarding, which is the only way to connect currently (since the service Grafana dashboard does not have an ingress entry on it's own)
```

Comment:
Ingress can be configured on a per-project basis. Let's mention it as an alternative for now.
```suggestion
You can connect to Grafana through local port forwarding using the steps below. Alternatively, you can set up ingress to point to Grafana.
```